types/jsonwebtoken/index.d.ts

```diff
@@ -107,15 +107,39 @@ export type SignCallback = (
     err: Error | null, encoded: string | undefined
 ) => void;
 
+// standard names https://www.rfc-editor.org/rfc/rfc7515.html#section-4.1
 export interface JwtHeader {
-    alg: string;
+    alg: string | Algorithm;
     typ?: string;
+    cty?: string;
+    crit?: Array<string | keyof JwtHeader>;
```

Comment:
should this be
```suggestion
    crit?: Array<string | Exclude<keyof JwtHeader, 'crit'>>;
```

Reply:
I'm not sure that it matters, since `string | keyof JwtHeader` reduces to `string`.

Reply:
True, hoped that there would be some IDE hinting benefit...
I'll add it in case it ever will be supported.